apollo-gradle-plugin/src/main/java/com/apollographql/android/gradle/ApolloExtension.java

```diff
@@ -1,8 +1,14 @@
 package com.apollographql.android.gradle;
 
+import groovy.lang.Closure;
+
+import java.util.HashMap;
+import java.util.Map;
+
 public class ApolloExtension {
   static final String NAME = "apollo";
   private boolean generateClasses = false;
+  private Map<String, String> customTypeMapping = new HashMap<>();
```

Comment:
my general nitpick  "LinkedHashMap is always better than Hashmap" https://publicobject.com/2016/02/08/linkedhashmap-is-always-better-than-hashmap/